apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -152,6 +153,14 @@ class CensusForm extends Component {
 
   processError(error) {
     console.log(JSON.stringify(error, null, 2));
```

Comment:
Not so long ago we'd avoid doing `console.log` because it would fail on older IE versions.  While they might not officially be supported by us anymore, is it perhaps useful to remove this one anyway?

Reply:
Agreed, now that we're actually handling the error from the server (and doing other validation on the client) I think it's best to remove this.